server/methods.js

```diff
@@ -267,6 +267,22 @@ Meteor.methods({
     if (this.userId) {
       let keys;
       const email = Meteor.users.findOne({ _id: this.userId }).emails[0].address;
+      const existingAccounts = Meteor.users.find({'emails.address': {'$in': mods}}).fetch().map(el => el.emails[0].address);
+      const newAccounts = mods.filter(email => !existingAccounts.includes(email));
+      newAccounts.forEach(async (item) => {
+        await Accounts.createUser({
+          email: item,
+          password: Math.random().toString(36).substr(2, 7),
+          profile: {
+            name: item.slice(0, item.indexOf('@'))
+          }
+        });
+        await Accounts.forgotPassword({email: item}, (err) => {
```

Comment:
@singhsanket143 `MAIL_ID_FROM` is not set here for some reason

Reply:
@peter-hank According to me in Accounts.forgotPassword only the target email whose password is supposed to be reset is passed, the MAIL_ID_FROM will be preconfigured from the code and session variables.

Reply:
@singhsanket143 unfortunately, it doesn't work, it doesn't use the `MAIL_ID_FROM` variable in this case

Reply:
@peter-hank I have updated the code to rectify this issue.